incubator/hnc/internal/validators/object_test.go

```diff
@@ -492,6 +493,33 @@ func TestCreatingConflictSource(t *testing.T) {
 		name:             "Allow creation of source objects with no conflict",
 		newInstName:      "secret-a",
 		newInstNamespace: "a",
+	}, {
+		name:              "Allow creation of source objects with treeSelector on the conflicting child",
+		forest:            "-aa",
+		conflictInstName:  "secret-b",
+		conflictNamespace: "b",
+		newInstName:       "secret-b",
+		newInstNamespace:  "a",
+		newInstAnnotation: map[string]string{api.AnnotationTreeSelector: "c"},
+		fail:              false,
+	}, {
+		name:              "Allow creation of source objects with selector on the conflicting child",
```

Comment:
Update this and the previous case names with `Allow creation of source objects with selector/treeSelector not matching the conflicting child`?

Reply:
done